config/base/params-vllm-cpu.env

```diff
@@ -0,0 +1 @@
+vllm-cpu-image=quay.io/modh/vllm:rhoai-v2-19-cpu
```

Comment:
@riprasad this should get automatically updated by konflux, right?

Reply:
Yes, we will use the digest and setup nudging for automated updates!

Reply:
umm, given this image will not be part of DIH. We can use the image tag and not digest and bypass the nudging for it!

We'll take care of the updates!